plugin/kubernetai/setup.go

```diff
@@ -39,7 +39,10 @@ func setup(c *caddy.Controller) error {
 
 // Parse parses multiple kubernetes into a kubernetai
 func Parse(c *caddy.Controller) (*Kubernetai, error) {
-	var k8i = &Kubernetai{}
+	var k8i = &Kubernetai{
+		autoPathSearch: searchFromResolvConf(),
```

Comment:
what happens if this returns `nil`?  Would this function be better to return `(string[], bool)` for `autoPathSearch, ok := searchFromResolvConf()` and deal with it not being found?

Reply:
nil is not a problem ,it's considered an empty array, so basically it will continue as if nothing happened.